doc/api/http.markdown

```diff
@@ -43,12 +43,6 @@ list like the following:
       'Host', 'mysite.com',
       'accepT', '*/*' ]
 
-## http.METHODS
-
-* {Array}
-
-A list of the HTTP methods that are supported by the parser.
-
```

Comment:
I don't see how standard verb like `MKCALENDAR` is any different from non-standard one like `HELLOWORLD`. Your application will likely respond the same way for both of them.

And since treating all uncommon methods as bad input is undesirable, we could just allow all methods.

Seriously, what is the benefit of enforcing such limits?